packages/commons-test/src/testUtils.ts

```diff
@@ -1096,3 +1098,29 @@ export const getMockContextM2M = ({
   logger: genericLogger,
   requestTimestamp: Date.now(),
 });
+
+export const getMockSessionClaims = (
+  role: string = userRole.ADMIN_ROLE
+): SessionClaims => ({
+  uid: generateId(),
+  organization: {
+    id: generateId(),
+    name: "My Org",
+    roles: [{ role }],
+  },
+  name: "A generic user",
+  family_name: "Family name",
+  email: "randomEmailforTest@tester.com",
+});
+
+export const getMockCustomClaims = (
```

Comment:
these claims are always part of the session token, you could merge them if you prefer

Reply:
good idea, this commit [c680964](https://github.com/pagopa/interop-be-monorepo/pull/1896/commits/c680964deccf86f4d1ea50ea381ad8b3031823cb) apply this suggestion, getMockCustomClaims is not more used.